ansible_collections/arista/cvp/plugins/modules/cv_change_control_v3.py

```diff
@@ -150,8 +150,10 @@ def main():
     argument_spec = dict(
         name=dict(type='str'),
         change=dict(type='dict'),
-        state=dict(default='show', type='str', choices=['show', 'set', 'remove']),
-        change_id=dict(type='list', elements='str')
+        state=dict(default='show', type='str', choices=['show', 'set', 'remove', 'approve', 'unapprove', 'execute',
```

Comment:
Can you update the docstrings above to reflect new `state` options?

Reply:
The how-to and docstrings have been updated as suggested to reflect the new options. The examples have also been expanded to include some of the new options.